okhttp-ws/src/main/java/com/squareup/okhttp/ws/WebSocketCall.java

```diff
@@ -172,32 +156,32 @@ private void createWebSocket(Response response, WebSocketListener listener)
 
   // Keep static so that the WebSocketCall instance can be garbage collected.
   private static class ConnectionWebSocket extends RealWebSocket {
```

Comment:
This might deserve a rename since it no longer operates on a `Connection`. Don't really care what name... `StreamWebSocket` maybe?